### PR TITLE
Resolve #95 - Update Oghliner app content

### LINKED ISSUE #95: redesign Oghliner website
The Oghliner website at https://mozilla.github.io/oghliner/ was designed using the GitHub Pages generator, and it could use a redesign (both graphic design and information design) in order to better convey what the project is about and how to go about using it.

This used to be identical to #7, because we were using the website as the template for bootstrapping new apps. But now we've separated those two things, so the website is in app/, while the _bootstrap_ command uses the template in templates/. Thus redesigning the website is now a separate concern from redesigning the template.

### PR DESCRIPTION
**Pull request by darkwing**
This is just a start, will be building upon this.

### CODE REVIEW

**Comment by darkwing** [comment]
This is ready for preliminary review.  I need to figure out a way to get rid of the buttons though.

**Comment by marco-c** [comment]
I like the idea of reusing the README (idea that made me remember that we need to keep the README updated, filed #153).
I also like the result. I'd only add a bit more space between the sections (e.g. before "Using The Template").

File: gulpfile.js [hunk 1]
```diff
@@ -21,14 +21,25 @@ var gulp = require('gulp');
...
 var oghliner = require('./index.js');
 
 gulp.task('default', ['build', 'offline']);
 
-gulp.task('build', ['copy-files', 'build-tabzilla']);
+gulp.task('build', ['build-app', 'build-tabzilla']);
+
+gulp.task('build-app', ['copy-files'], function() {
+  return gulp.src('dist/index.html')
+              .pipe(template({
```
Comment (marco-c) [inline gulpfile.js]: Nit: To keep the same style as the rest of the file:

```
  return gulp.src('dist/index.html')
    .pipe(template({
    ...
```

**Comment by marco-c** [comment]
I also think the information could be reorganized a little in the README, to make it simpler to read.

**Comment by darkwing** [comment]
Updates made to separate sections and update code style.

**Comment by marco-c** [comment]
Last remaining issue:
screen

**Comment by darkwing** [comment]
I tried `white-space: pre-line` but we were still getting overflow so the best solution is hiding overflow so code/commands look the same and we never see screen stretching.

**Comment by marco-c** [comment]
Can we use `overflow: auto` so the scrollbars are only shown if needed?

**Comment by darkwing** [comment]
/me facepalms

Of course, sorry.  No more early morning PRs from me

**Comment by marco-c** [comment]
:smile:

**Pull request merged**